action.yml

```diff
@@ -235,26 +235,28 @@ runs:
       shell: bash
       working-directory: repository
       run: |
-        git config user.name searchspring-machine
-        git config user.email machine@searchspring.com
+        if [ ${{ steps.variables.outputs.branch }} = production ]; then
```

Comment:
Will this happen on both push and pull request runs?

Reply:
Nope, `${{ steps.variables.outputs.branch }}` is only ever `production` on the merge of the PR into production. The PR run contains the branch being merged

Reply:
Cool - just wanted to be sure :)